test/Driver/sdk-apple.swift

```diff
@@ -34,45 +34,37 @@
 // RUN: %empty-directory(%t/MacOSX10.12.Internal.sdk) && not %swift_driver -sdk %t/MacOSX10.12.Internal.sdk -### 2>&1 | %FileCheck -check-prefix=SDK-TOO-OLD %s
 // RUN: %empty-directory(%t/MacOSX10.13.sdk) && not %swift_driver -sdk %t/MacOSX10.13.sdk -### 2>&1 | %FileCheck -check-prefix=SDK-TOO-OLD %s
 // RUN: %empty-directory(%t/MacOSX10.13.Internal.sdk) && not %swift_driver -sdk %t/MacOSX10.13.Internal.sdk -### 2>&1 | %FileCheck -check-prefix=SDK-TOO-OLD %s
-// RUN: %empty-directory(%t/MacOSX10.14.sdk) && %swift_driver -sdk %t/MacOSX10.14.sdk -### 2>&1 | %FileCheck -check-prefix=SDK-OKAY %s
-// RUN: %empty-directory(%t/MacOSX10.14.Internal.sdk) && %swift_driver -sdk %t/MacOSX10.14.Internal.sdk -### 2>&1 | %FileCheck -check-prefix=SDK-OKAY %s
+// RUN: %empty-directory(%t/MacOSX10.14.sdk) && not %swift_driver -sdk %t/MacOSX10.14.sdk -### 2>&1 | %FileCheck -check-prefix=SDK-TOO-OLD %s
+// RUN: %empty-directory(%t/MacOSX10.14.Internal.sdk) && not %swift_driver -sdk %t/MacOSX10.14.Internal.sdk -### 2>&1 | %FileCheck -check-prefix=SDK-TOO-OLD %s
+// RUN: %empty-directory(%t/MacOSX10.15.sdk) && %swift_driver -sdk %t/MacOSX10.15.sdk -### 2>&1 | %FileCheck -check-prefix=SDK-OKAY %s
+// RUN: %empty-directory(%t/MacOSX10.15.Internal.sdk) && %swift_driver -sdk %t/MacOSX10.15.Internal.sdk -### 2>&1 | %FileCheck -check-prefix=SDK-OKAY %s
 // RUN: %empty-directory(%t/OSX50.sdk) && %swift_driver -sdk %t/OSX50.sdk -### 2>&1 | %FileCheck -check-prefix=SDK-OKAY %s
 
-// RUN: not %swift_driver -sdk %t/MacOSX10.9.sdk/ -### 2>&1 | %FileCheck -check-prefix=SDK-TOO-OLD %s
-// RUN: not %swift_driver -sdk %t/MacOSX10.9.Internal.sdk/ -### 2>&1 | %FileCheck -check-prefix=SDK-TOO-OLD %s
-// RUN: not %swift_driver -sdk %t/MacOSX10.10.sdk/ -### 2>&1 | %FileCheck -check-prefix=SDK-TOO-OLD %s
-// RUN: not %swift_driver -sdk %t/MacOSX10.10.Internal.sdk/ -### 2>&1 | %FileCheck -check-prefix=SDK-TOO-OLD %s
-// RUN: not %swift_driver -sdk %t/MacOSX10.11.sdk/ -### 2>&1 | %FileCheck -check-prefix=SDK-TOO-OLD %s
-// RUN: not %swift_driver -sdk %t/MacOSX10.11.Internal.sdk/ -### 2>&1 | %FileCheck -check-prefix=SDK-TOO-OLD %s
-// RUN: not %swift_driver -sdk %t/MacOSX10.12.sdk/ -### 2>&1 | %FileCheck -check-prefix=SDK-TOO-OLD %s
-// RUN: not %swift_driver -sdk %t/MacOSX10.12.Internal.sdk/ -### 2>&1 | %FileCheck -check-prefix=SDK-TOO-OLD %s
-// RUN: not %swift_driver -sdk %t/MacOSX10.13.sdk/ -### 2>&1 | %FileCheck -check-prefix=SDK-TOO-OLD %s
-// RUN: not %swift_driver -sdk %t/MacOSX10.13.Internal.sdk/ -### 2>&1 | %FileCheck -check-prefix=SDK-TOO-OLD %s
-// RUN: %swift_driver -sdk %t/MacOSX10.14.sdk/ -### 2>&1 | %FileCheck -check-prefix=SDK-OKAY %s
-// RUN: %swift_driver -sdk %t/MacOSX10.14.Internal.sdk/ -### 2>&1 | %FileCheck -check-prefix=SDK-OKAY %s
```

Comment:
As far as I can tell this was doing the same thing as the above checks. Maybe it was looking for just "OSX" at one point and then we dropped that?